File: Divyansh Singh/Array/find the highest altitude.cpp
class Solution {
public:
    int largestAltitude(vector<int>& gain) {
        int maxi=0,sum=0;
        for(int i=0;i<gain.size();i++){
            sum=sum+gain[i];
            maxi=max(maxi,sum);
        }
        return maxi;
    }
};

File: Khushboo Singh/Array/Find the Highest Altitude.java
class Solution {
    public int largestAltitude(int[] gain) {
        int n=gain.length,max=0,sum=0;
        for(int i=0;i<n;i++){
            sum+=gain[i];
            max=Math.max(max,sum);
        }
        return max;
    }
}
